packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable valid-jsdoc */
+
+import Batcher from '../batcher';
+
+const CallDiagnosticEventsBatcher = Batcher.extend({
+  namespace: 'Metrics',
+
+  /**
+   * @param webClientDomain
+   * @returns
+   */
+  getBuildType(webClientDomain) {
+    if (
+      webClientDomain?.includes('teams.webex.com') ||
```

Comment:
teams.webex.com is not used anymore, it just redirects to web.webex.com, right? so this line is not needed

Reply:
I can remove it, this was just a copy paste from old helper functions.